website/docs/BasicInterfaces/Agent.md

```diff
@@ -74,7 +74,7 @@ const { Agent } = require('secret-agent');
   - connectionToCore `options | ConnectionToCore`. An object containing `IConnectionToCoreOptions` used to connect, or an already created `ConnectionToCore` instance. Defaults to automatically booting up and connecting to a local `Core`.
   - name `string`. This is used to generate a unique sessionName.
   - userAgent `strong`. This sets your browser's user agent string. Prefixing this string with a tilde (~) allows for dynamic options.
-  - browserEmulatorId `string`. Emulates the unique properties that help SecretAgent look like a normal browser.
+  - browserEmulatorId `string`. Emulates the properties that help SecretAgent look like a normal browser.
```

Comment:
There's a more detailed description under "handler" that seems to give a bit more info on what to actually do with this

Reply:
Fixed